.github/workflows/images.yaml

```diff
@@ -62,6 +64,7 @@ jobs:
       - name: Build / Push
         uses: docker/build-push-action@v3
         with:
+          context: .
```

Comment:
I thought this was the default value, right? 🤔

Reply:
No. It default use git content. It will ignore `.git` dir. So git can't get the tag when build docker image.

Reply:
Ahhh that's great to know! Thanks for finding it out and sharing your solution ❤️

For anyone interested, it's documented in the build-push-action doc:

https://github.com/docker/build-push-action#git-context